scalatest/src/main/scala/org/scalatestplus/junit/AssertionsForJUnit.scala

```diff
@@ -96,7 +95,7 @@ import org.scalactic._
  */
 trait AssertionsForJUnit extends Assertions {
 
-  private[scalatest] override def newAssertionFailedException(optionalMessage: Option[String], optionalCause: Option[Throwable], pos: source.Position): Throwable = {
+  private[org] override def newAssertionFailedException(optionalMessage: Option[String], optionalCause: Option[Throwable], pos: source.Position): Throwable = {
     new JUnitTestFailedError(optionalMessage, optionalCause, pos, None)
```

Comment:
Instead of private[org] we should make a copy of these and move them to the scalatestplus.* packages where they are needed. Eventually the stuff in the scalatestplus.* packages will go to a separate github repo.